docs/source/reference/aot-compilation.rst

```diff
@@ -29,6 +29,17 @@ Ahead-of-Time compilation
       be written to.  By default this follows the Python naming convention
       for the current platform.
 
+   .. attribute:: target_cpu
+
+      (read-write attribute) The name of the CPU model to generate code for.
+      This will select the appropriate instruction set extensions.  By
+      default, a generic CPU is selected in order to produce portable code.
+
+      Recognized names for this attribute depend on the current architecture
+      and LLVM version.  If you have LLVM installed, ``llc -mcpu=help``
+      will give you a list.  You can also give the value ``"host"`` which
+      will select the current host CPU.
```

Comment:
Can we list some example names that cover common cases, for users who don't have llc handy?  (Since you don't need it to install Numba.)  For example, `x86-64`, `ivybridge`, `haswell`, `broadwell`, `skylake` should cover most things someone wants to do...